app/models/invenio_rdm_record_converter.rb

```diff
@@ -77,7 +77,9 @@ def invenio_provenance(proxy_depositor, on_behalf_of)
   def invenio_metadata(gf)
     {
       "resource_type": resource_type(gf.resource_type.shift),
-      "creators": creators(gf.creator)
+      "creators": creators(gf.creator),
+      "title": gf.title.first,
+      "additional_titles": gf.title.last(gf.title.size-1).map{ |title| {"title": title, "type": "alternative_title", "lang": "eng"} }
```

Comment:
Looks pretty straightforward to me.

Reply:
yeah, it was. I think I forgot to rebase this previously, but now it has been. Merging now.